sdk/function_id_test.go

```diff
@@ -0,0 +1,50 @@
+package sdk
+
+import (
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"github.com/capeprivacy/cli/entities"
+)
+
+func testServer(t *testing.T) *httptest.Server {
+	t.Helper()
+	// a server that returns deployment info
+	s := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		deployment := entities.Deployment{
+			ID:                  "abc123",
+			UserID:              "github123",
+			Name:                "isprime",
+			Location:            "mars",
+			AttestationDocument: nil,
+			CreatedAt:           time.Time{},
+		}
+
+		if err := json.NewEncoder(w).Encode(deployment); err != nil {
+			t.Fatal(err)
+
+			return
+		}
+
+		w.WriteHeader(http.StatusOK)
+	}))
+
+	t.Cleanup(s.Close)
+
+	return s
+}
+
+func TestFunctionID(t *testing.T) {
```

Comment:
We could probably have more test cases .. e.g., what happens if the server returns a 400? Or a 500?